main.nf

```diff
@@ -1,31 +1,34 @@
 // Ensure DSL2
 nextflow.enable.dsl = 2
 
+params.s3_file = "s3://iatlas-project-tower-bucket/s3_file.csv"
 //url for example CWL workflow
-params.cwl_file = "https://raw.githubusercontent.com/CRI-iAtlas/iatlas-workflows/develop/EPIC/workflow/steps/epic/epic.cwl"
-//example params file
-params.input_file = "${projectDir}/example_inputs/epic.json"
+params.cwl_file = "https://raw.githubusercontent.com/CRI-iAtlas/iatlas-workflows/develop/Immune_Subtype_Clustering/workflow/steps/immune_subtype_clustering/immune_subtype_clustering.cwl"
```

Comment:
In the future, we can include the cwl_file as a field in the input CSV file, and therefore we can execute any number of different CWL workflows in a single run if we want to. Just wanted to get this thought down here, but I did not think it was necessary to implement for this task.